api/swagger/header.json

```diff
@@ -17,5 +17,13 @@
     {
       "basicAuth": []
     }
-  ]
+  ],
```

Comment:
@PavelKhripkov Nice change! I understand this will apply the language settings to all endpoints, right?